packages/client/components/ActionSidebarAgendaItemsSection.tsx

```diff
@@ -1,32 +1,48 @@
 import graphql from 'babel-plugin-relay/macro'
-import React from 'react'
+import React, {useMemo} from 'react'
 import {createFragmentContainer} from 'react-relay'
 
 import {ActionSidebarAgendaItemsSection_meeting} from '../__generated__/ActionSidebarAgendaItemsSection_meeting.graphql'
 import useGotoStageId from '../hooks/useGotoStageId'
 import AgendaListAndInput from '../modules/teamDashboard/components/AgendaListAndInput/AgendaListAndInput'
 import MeetingSidebarPhaseItemChild from './MeetingSidebarPhaseItemChild'
+import {NavSidebar} from '../types/constEnums'
 
 interface Props {
   gotoStageId: ReturnType<typeof useGotoStageId>
   handleMenuClick: () => void
   meeting: ActionSidebarAgendaItemsSection_meeting
+  maxAgendaItemsHeight: number
 }
 
 const ActionSidebarAgendaItemsSection = (props: Props) => {
-  const {gotoStageId, handleMenuClick, meeting} = props
-  const {team} = meeting
+  const {gotoStageId, handleMenuClick, meeting, maxAgendaItemsHeight} = props
+  const {phases, team} = meeting
   const handleClick = async (stageId: string) => {
     gotoStageId(stageId).catch()
     handleMenuClick()
   }
   // show agenda (no blur) at all times if the updates phase isNavigable
   // facilitator can click on updates nav item before completing all check-in stages
-  const updatesPhase = meeting.phases!.find((phase) => phase.phaseType === 'updates')!
+  const updatesPhase = phases.find((phase) => phase.phaseType === 'updates')
   const isUpdatesNavigable = updatesPhase && updatesPhase.stages![0].isNavigable
+  const agendaItemsCount = useMemo(() => {
```

Comment:
Fetching and filtering the `agendaItems` here and a few children below doesn't feel great but as `agendaItems` can live in `teams` or the `meeting`, I think it's necessary